README.md

```diff
@@ -90,7 +90,7 @@ You can also run Dartagnan from the console:
 ```
 java -jar dartagnan/target/dartagnan-3.1.1.jar <CAT file> [--target=<arch>] <program file> [options]
 ```
-For programs written in `.c` and `.bpl`, value `<arch>` specifies the programming language or architectures to which the program will be compiled. Program written in `.litmus` format do not require such option. `<arch>` must be one of the following: 
+For programs written in `.c` and `.bpl`, value `<arch>` specifies the programming language or architectures to which the program will be compiled. For program written in `.litmus` format, if the `--target` option is not given, dartagnan will automatically extract the `<arch>` from the litmus test header. `<arch>` must be one of the following: 
```

Comment:
Typo: "For program_s_ written in ..."
Also, I think we should capitalize "Dartagnan" since it is a proper name.